partcad-cli/src/partcad_cli/click/commands/import.py

```
@@ -0,0 +1,46 @@
+
+    name = Path(existing_part).stem
+
+    config = {"desc": desc} if desc else {}
+
+    try:
+        import_part_action(project, part_type, name, existing_part, config, target_format)
+        pc_logging.info(f"Successfully imported part: {name}")
+        click.echo(f"Part '{name}' imported successfully.")
+    except Exception as e:
+        pc_logging.exception(f"Error importing part '{name}' ({part_type})")
+        raise click.ClickException(f"Error importing part '{name}' ({part_type}): {e}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)